src/common/marlin_server.cpp

```diff
@@ -987,14 +988,17 @@ static uint64_t _server_update_vars(uint64_t update) {
     }
 
     if (update & MARLIN_VAR_MSK(MARLIN_VAR_SD_PDONE)) {
-        uint8_t progress = 0;
-        if (oProgressData.oPercentDone.mIsActual(marlin_server.vars.print_duration))
-            progress = static_cast<uint8_t>(oProgressData.oPercentDone.mGetValue());
-        else
-            progress = static_cast<uint8_t>(media_print_get_percent_done());
-        if (marlin_server.vars.sd_percent_done != progress) {
-            marlin_server.vars.sd_percent_done = progress;
-            changes |= MARLIN_VAR_MSK(MARLIN_VAR_SD_PDONE);
+        if (!FirstLayer::isPrinting()) { /// push notifications used for first layer calibration
```

Comment:
This is to be done only when NOT printing? Are you sure this won't break the printing progress reporting? Previous code didn't have this limitation...

Reply:
1st layer calib. is not printing from SD/USB so this has to be disabled in case of 1st layer calib. pattern printing.

Reply:
ok